gradle.properties

```diff
@@ -8,8 +8,8 @@
 #     http://www.apache.org/licenses/LICENSE-2.0
 #
 ### dependencies
-pravegaVersion=0.4.0-50.63aea88-SNAPSHOT
-flinkConnectorVersion=0.4.0-103.7971206-SNAPSHOT
+pravegaVersion=0.4.0-1936.4441cb0-SNAPSHOT
+flinkConnectorVersion=0.4.0-126.7971206-SNAPSHOT
```

Comment:
I believe both Pravega and Connector versions are not published yet in central repository. Merging the code will break the build.

Reply:
I think that the Pravega artifact is ready with this fix in `jFrog`, but I don't know about the Flink connector. Please, let me know when the Flink connector artifact is ready to update this.